packages/backend-app-api/config.d.ts

```diff
@@ -88,6 +88,47 @@ export interface Config {
                */
               subject: string;
             };
+            /**
+             * The scope of access permitted for this access method. If no scope
+             * is given, it'll have unlimited access. This access restriction
+             * applies for the framework level; individual plugins may have
+             * their own access control mechanisms on top of this.
+             */
+            scope?: {
```

Comment:
I'm thinking `restrictions` here might be a better name? "scope" (in OAuth) is generally used for additions and extending the scope of what a token is authorized for. This is kind of the opposite, where we restrict access instead.

Reply:
Oh and an addon - maybe the wording "where we restrict access" isn't how I think of it at least, this field is how we GRANT access - but the caveat is, the default value is "all" so to speak. If the default value weren't "all", then "scope" might be less confusing right? As a matter of fact, maybe that's a discussion to have; what SHOULD the default policy be?

Reply:
I think allowing all as a default policy is pretty sensible tbh, but I agree it gets a bit tricky where we start by restricting but then expand. Sill feeling that it makes sense with the example from below with either `restrictions` or `accessRestrictions`:

```yaml
externalAccess:
  - type: static
    options: { ... }
    accessRestrictions:
      - plugin: scaffolder
      - plugin: catalog
        permissionAttributes: { action: read }
```